ansible/roles/products/common/default-icon/defaults/main.yml

```diff
@@ -16,4 +16,8 @@
 user: "{{ ansible_user_id }}"
 user_folder: "/home/{{ user }}"
 shadow_hand_launcher_folder: "{{ user_folder }}/.shadow_launcher_app_{{ container_name }}/shadow_hand_launcher"
-haptx_mapping_script: ""
\ No newline at end of file
+haptx_mapping_script: ""
+enable_warning: false
+warning_title: "Example Warning"
+warning_text: "Example Text"
```

Comment:
Do we want to keep the "Example" in both title and text?

Reply:
Yeah, as we still need this value as we share icons between products, so it will never be activated with other products, but it will need to be present with a default value so it can fill in the template. I think having example text is better than just having it appear as blank